libraries/SITL/SIM_JEDEC_MX25L3206E.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "SIM_JEDEC.h"
+
+namespace SITL {
+
+class JEDEC_MX25L3206E : public JEDEC
+{
+protected:
+
+    void fill_rdid(uint8_t *buf, uint8_t len) override;
+    void fill_rdsr(uint8_t *buf, uint8_t len) override;
+
+    const char *filename() const override { return "dataflash.bin"; }
+
+    uint32_t storage_size() const override {
+        // page size * page per block * no. of blocks = 256 * 256 * 64 = 4194304
+        return 4194304;
+    }
```

Comment:
Why not do this in the parent class, or as a `constexpr` based on the othe callbacks?

Reply:
Moved to parent class